contrib/ci/Dockerfile-debian

```diff
@@ -0,0 +1,38 @@
+FROM debian:testing
+
+RUN echo "deb http://ftp.us.debian.org/debian unstable main contrib non-free" >> /etc/apt/sources.list
+RUN echo 'Package: *\n\
+Pin: release a=testing\n\
+Pin-Priority: 900\n\
+\n\
+Package: *\n\
+Pin: release a=unstable\n\
+Pin-Priority: 800\n'\
+ > /etc/apt/preferences
+ > /etc/apt/preferences
+RUN cat /etc/apt/preferences
+RUN echo fubar > /etc/machine-id
+
+RUN apt-get update -qq
+RUN apt-get install -yq --no-install-recommends \
+	gnutls-bin \
+	gnutls-dev \
+	gobject-introspection \
+	gtk-doc-tools \
+	libgirepository1.0-dev \
+	libglib2.0-bin \
+	libglib2.0-dev \
+	libglib2.0-dev-bin \
+	libgpgme11-dev \
+	libjson-glib-dev \
+	ninja-build \
+	pkg-config \
+	python3-distutils \
+	python3-pip \
+	python3-setuptools \
+	python3-wheel
+
+# Meson is too old in unstable, and that won't change until Buster is released
+RUN pip3 install meson
```

Comment:
Is this true?  What version of meson do you need?

Reply:
0.47.0 -- tbh I don't really know what `unstable` and `testing` are...

Reply:
🤦‍♂ Here's a decoder ring:
Unstable: brand new stuff
Testing: brand new stuff that has passed automatic testing and promption after X days
Stable: Current release

https://tracker.debian.org/pkg/meson

It's new enough in testing and stable.